Formula/yarn.rb

```diff
@@ -18,9 +16,10 @@ class Yarn < Formula
   depends_on "node"
 
   def install
-    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
-    bin.install_symlink Dir["#{libexec}/bin/*"]
-    inreplace "#{libexec}/lib/node_modules/yarn/package.json", '"installationMethod": "tar"', '"installationMethod": "homebrew"'
+    libexec.install Dir["*"]
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarn"
+    bin.install_symlink "#{libexec}/bin/yarn.js" => "yarnpkg"
```

Comment:
```sh-session
$ readlink /usr/local/bin/yarn
../Cellar/yarn/0.21.3/bin/yarn
$ realpath /usr/local/bin/yarn
/usr/local/Cellar/yarn/0.21.3/libexec/bin/yarn.js
```

Can `realpath` be trusted to exist on the platforms that runs that shellscript?

Although `yarnpkg` just runs `yarn.js`, it doesn't do the stuff `yarn` does. Is that by design?